src/app/pages/Home/components/MovieItem/MovieItem.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
+import ThreeDotMenu from "./../../../shared/ThreeDotMenu/ThreeDotMenu";
```

Comment:
since you have this line in your `webpack` config:
```
modules: [path.resolve(__dirname, 'src'), 'node_modules'],
```

you can use absolute paths for your import components to avoid this long relative ones.
Try something like this: `import ThreeDotMenu from "app/pages/shared/ThreeDotMenu/ThreeDotMenu";` and see whether it works or not

Reply:
Yes, it is working, thank you for the recommendation.